src/base/bittorrent/session.cpp

```diff
@@ -3091,6 +3092,24 @@ void Session::setIPFilterFile(const Path &path)
     }
 }
 
+bool Session::isExcludedFileNamesEnabled() const
+{
+    return m_isExcludedFileNamesEnabled;
+}
+
+void Session::setExcludedFileNamesEnabled(const bool enabled)
+{
+    if (m_isExcludedFileNamesEnabled == enabled)
+        return;
+
+    m_isExcludedFileNamesEnabled = enabled;
+
+    if (enabled)
+        populateExcludedFileNamesRegExpList();
+    else
+        m_excludedFileNamesRegExpList.clear();
```

Comment:
Not sure if this is needed.

Reply:
This line is not needed.

Reply:
> This line is not needed.

Agree.


> Actually this `if (enabled) ...` is also not needed.

Why to re-populate regex list once user disabled the feature?

Reply:
> > This line is not needed.
> 
> Agree.

On the other hand, why leave it populated if it will still be re-populated when it is enabled again.

Reply:
>Actually this if (enabled) ... is also not needed.

How will the list be populated if qBt starts with this option disabled but user wants to enable it without changing the filters list?

Awaiting your decision.

Reply:
Disregard this discussion, leave this function as is.